apps/fabric-website/src/pages/Styles/OfficeBrandIconsPage/OfficeBrandIconsPage.tsx

```diff
@@ -173,7 +170,7 @@ function _otherSections(platform: Platforms): IPageSectionProps<Platforms>[] {
             <>
               <ul className={styles.iconList}>
                 {productIcons.map((icon, iconIndex) => (
-                  <li key={iconIndex}>
+                  <li key={icon.icon}>
```

Comment:
Should this be:

```suggestion
                  <li key={icon.name}>
```

Reply:
Doesn't really matter since either one will be unique in this case (these are a custom type not IIconProps)